src/mds/MDCache.h

```diff
@@ -1215,6 +1222,8 @@ class MDCache {
   StrayManager stray_manager;
 
  private:
+  std::set<inodeno_t> replay_taked_inos; // the inos have been taken when replaying
```

Comment:
nit: "taked" is not a word in English. Can you change this to "taken" ?

Reply:
Done, thanks :-)